src/LIBSVM.jl

```diff
@@ -144,7 +143,13 @@ function svmmodel(mod::SVM)
         sv_coef[i] = pointer(mod.coefs, (i-1)*n+1)
     end
 
-    nodes, ptrs = LIBSVM.instances2nodes(mod.SVs.X)
+    if !isa(mod.kernel, Kernel.KERNEL) || mod.kernel == Kernel.Precomputed
```

Comment:
I think we can simply have this?
```suggestion
    if kernel == Int32(Kernel.Precomputed)
```